doc/engineering-reference/src/simulation-models-encyclopedic-reference-004/system-availability-managers.tex

```diff
@@ -276,6 +276,18 @@ \subsubsection{Dew-Point control}\label{dew-point-control}
 
 If the actual zone air humidity ratio is above the humidity ratio setpoint and the controlled zone needs dehumidification as the second scenario, then the initial decision is overridden and natural ventilation is not allowed. This is intended to avoid humidifying a space, which could result in additional dehumidification load.
 
+\subsubsection{Operative Temperature Control with 80% Acceptability Limits}\label{operative-temperature-control-with-80-acceptability-limits}
```

Comment:
@mjwitte You are right. The % sing need a backslash to display correct. Unfortunately, the backslash work in the text section and does not work inside the subsubsection. Instead of using % sign, I user "Percent" inside the subsubsection. Here are my change:

![capture](https://cloud.githubusercontent.com/assets/5970656/22757360/6fd128ac-ee18-11e6-88dc-a6efed059dff.JPG)